public/viewforum.php

```diff
@@ -134,7 +134,7 @@
 
         echo "<tr>";
 
-        echo "<td class='p-1'><img src='" . asset('assets/images/icon/forum-topic-unread.gif') . "' width='20' height='20' title='No unread posts' alt='No unread posts'></td>";
+        echo "<td class='p-1'>" . Blade::render('<x-fas-arrow-alt-circle-right class="h-4 w-4" :altText="$altText"/>', ['altText' => "No unread posts"]) . "</td>";
```

Comment:
`alttext` is not a valid HTML attribute. Recommend instead using `title` on the `<td />` element.

Reply:
Sounds good.

For the record: as is, the code seems to use the term `No unread posts` as the `alt` for this icon on all topics, regardless of whether they are actually read or unread. I went into the repo history and it's been there ever since the birth of the repo (first commit that included the viewforum.php file: https://github.com/RetroAchievements/RAWeb/commit/83abe6097135bdef24bd2a0740444460b49772f3)

We should probably use a different `title`, unless you know somewhere in the code that modifies these via JS or something (me searches came up empty).

`title="Forum topic"` and `title="Forum` for `viewforum.php` and `forum.php`, respectively?

Thoughts?

Reply:
Good catch. In that case, it may actually be harmful to ascribe any semantic meaning to these icons. Maybe we should just set them to `aria-hidden="true"`.

Reply:
Yeah I had that thought as well. Sounds good!